testsuite/pytests/test_sp/test_sp_manager.py

```diff
@@ -114,12 +116,12 @@ def test_getting_kernel_status(self):
                             )
 
         sp_synapses = nest.structural_plasticity_synapses['syn1']
-        assert ('pre_synaptic_element' in sp_synapses)
-        assert ('post_synaptic_element' in sp_synapses)
-        assert (sp_synapses['pre_synaptic_element'] == 'Axon_ex')
-        assert (sp_synapses['post_synaptic_element'] == 'Den_ex')
+        assert 'pre_synaptic_element' in sp_synapses
+        assert 'post_synaptic_element' in sp_synapses
+        assert sp_synapses['pre_synaptic_element'] == 'Axon_ex'
+        assert sp_synapses['post_synaptic_element'] == 'Den_ex'
 
-        assert (nest.structural_plasticity_update_interval == 10000.)
+        assert nest.structural_plasticity_update_interval == 10000.
```

Comment:
```suggestion
        assert nest.structural_plasticity_update_interval == 10000.0
```

Reply:
Pretty please? With sugar on top? :laughing:

Reply:
There can be no arguing in matters of taste. Committed!